account_credit_control/README.rst

```diff
@@ -16,14 +21,24 @@ You are able to specify a particular policy for one partner or one invoice.
 Usage
```

Comment:
Please add some information here in configuration about accounts tab in every policy because if user doesn't enter any account, policy per partner can't be selected.